cirq-google/cirq_google/engine/engine.py

```diff
@@ -780,11 +781,25 @@ def get_processor(self, processor_id: str) -> engine_processor.EngineProcessor:
         """
         return engine_processor.EngineProcessor(self.project_id, processor_id, self.context)
 
+    @deprecated(deadline="v1.0", fix="Use get_sampler instead.")
     def sampler(
         self, processor_id: Union[str, List[str]], gate_set: Serializer
     ) -> engine_sampler.QuantumEngineSampler:
         """Returns a sampler backed by the engine.
 
+        Args:
+            processor_id: String identifier, or list of string identifiers,
+                determining which processors may be used when sampling.
+            gate_set: Determines how to serialize circuits when requesting
```

Comment:
Might want to do a docs link here out to `cg.Serializer` so that the link shows up in the docstring. Also missing a returns block